src/jloptions.c

```diff
@@ -171,7 +171,8 @@ static const char opts[]  =
     " -i, --interactive                             Interactive mode; REPL runs and\n"
     "                                               `isinteractive()` is true.\n"
     " -q, --quiet                                   Quiet startup: no banner, suppress REPL warnings\n"
-    " --banner={yes|no|short|auto*}                 Enable or disable startup banner\n"
+    " --banner={yes|no|auto*|<size>}                Enable or disable startup banner, or specify a\n"
+    "                                               preferred <size> (tiny, short, or narrow).\n"
```

Comment:
Can't one also specify `full` here?
```suggestion
    "                                               preferred <size> (tiny, short, narrow, or full).\n"
```

Reply:
Indeed! I just missed it when writing it out there, nice catch :)